spec/rubocop/cop/style/hash_syntax_spec.rb

```diff
@@ -135,6 +135,12 @@
         new_source = autocorrect_source('{ :a=>1, :b=>2 }')
         expect(new_source).to eq('{ a: 1, b: 2 }')
       end
+
+      # Bug: https://github.com/bbatsov/rubocop/issues/5019
+      it 'auto-corrects a missing space when hash is used as argument' do
+        new_source = autocorrect_source('foo:bar => 1')
```

Comment:
I really hope we have some lint for this crazy, but valid, syntax.

Reply:
We do indeed. `Layout/SpaceBeforeFirstArg` normally takes care of this. 🙂